compiler/rustc_hir/src/definitions.rs

```diff
@@ -107,6 +107,9 @@ pub struct Definitions {
 
     /// Item with a given `LocalDefId` was defined during macro expansion with ID `ExpnId`.
     expansions_that_defined: FxHashMap<LocalDefId, ExpnId>,
+
+    /// The [StableCrateId] of the local crate.
+    stable_crate_id: StableCrateId,
```

Comment:
Where is this used?

Reply:
In a debug assert in `local_def_path_hash_to_def_id`.

Reply:
Yes, it was used more in the initial version of the PR, when the whole concept of `StableCrateId` was still very experimental. It doesn't hurt to have the assertion though.